doc/install_generic.inc

```diff
@@ -9,6 +9,8 @@ Installation
 Stable Installation
 -------------------
 
+.. comment: install_theano_stable_version
```

Comment:
You may not need the "`comment: `" part, I think every line starting with `..` that is not a recognized command will be treated as a comment.